client/lib/accept/test/index.js

```diff
@@ -49,11 +49,14 @@ describe( '#accept()', function() {
 
 	it( 'should clean up after itself once the prompt is closed', function( done ) {
 		accept( 'Are you sure?', function() {
-			process.nextTick( function() {
+			// HACK: This was originally called with `process.nextTick` instead of `setTimeout`
```

Comment:
i reverted both this file and `client/lib/accept/dialog.jsx` and the test still failed for me. Trying to figure out why now

Reply:
whats bad about this hack is that it forces the unit tests to actually take 200ms longer.  We should strive for each unit test to only take a few ms to run.

By using jest-timers I was able to remove the real timeout.

Reply:
```js
	it( 'should clean up after itself once the prompt is closed', function( done ) {
		accept( 'Are you sure?', function() {
			const timeout = 199;
			setTimeout( function() {
				expect( document.querySelector( '.accept-dialog' ) ).to.be.null;

				done();
			}, timeout );
			jest.runTimerToTime( timeout + 1 );
		} );
		document.querySelector( '.button.is-primary' ).click();
		try {
			jest.runAllTimers();
		} catch ( err ) {}
```

Reply:
I was able to simplify the hack to the following:

```javascript
accept( 'Are you sure?', function() {
	jest.useFakeTimers();
	process.nextTick( function() {
		expect( document.querySelector( '.accept-dialog' ) ).to.be.null;
	} );
	jest.runAllTimers();
} );
```